packages/router/package.json

```diff
@@ -16,8 +16,5 @@
     "@penumbra-zone/storage": "workspace:*",
     "@penumbra-zone/transport": "workspace:*",
     "@penumbra-zone/wasm": "workspace:*"
-  },
-  "devDependencies": {
-    "@types/chrome": "0.0.260"
```

Comment:
Noticed these types deps in nearly all packages. Moved to top level .package.json,